src/Common/src/Common/Reflection/ReflectionHelpers.cs

```diff
@@ -244,6 +249,8 @@ public static Type FindType(string[] assemblyNames, string[] typeNames)
     /// </returns>
     public static Type FindType(Assembly assembly, string typeName)
     {
+        ArgumentGuard.NotNull(assembly);
```

Comment:
Yeah, these are the ones that got flagged when S3900 was turned on. Clearly the analyzer doesn't catch everything, and since I didn't _want_ to change anything here I only did the minimum to clear the errors (since we're going to be closely reviewing this code for real soon)